src/test/java/com/databricks/spark/xml/JavaXmlSuite.java

```diff
@@ -37,8 +40,10 @@
     private String tempDir = "target/test/xmlData/";
 
     @Before
-    public void setUp() {
-        sqlContext = new SQLContext(new SparkContext("local[2]", "JavaXmlSuite"));
+    public void setUp() throws IOException {
+        SparkConf conf = new SparkConf().set("spark.sql.warehouse.dir",
+                Files.createTempDirectory("spark-warehouse").toString());
```

Comment:
Is this change required?

Reply:
I worked on both Mac and Windows and Spark 2.0.0 is affected by this issue on Windows :

java.net.URISyntaxException: Relative path in absolute URI: file:C:/Users/XXXXX/IdeaProjects/spark-xml/spark-warehouse

see https://issues.apache.org/jira/browse/SPARK-15893

This is the known workaround for this issue. I added this configuration to be able to build the project on all platforms.

Reply:
Okay, let's leave some comments and make this double-space indentation.

Reply:
Comment added.
This is a java file, it s indented with 4 spaces.